doc/design/vic-machine.md

```diff
@@ -179,6 +179,47 @@ Some of the elevated privilege operations could be delegated during self-provisi
 
 At this point install transitions to managing - reporting VCH status from initial install is the same as reporting that information for any VCH regardless of age.
 
+
+## Deleting - per vSphere target
+
+### Inputs
+
+1. vSphere SDK endpoint
+2. vSphere administrative credentials
+3. VCH identification
+ - VCH resource pool path in govc format, e.g. /ha-datacenter/vm/vch1
+
+### Actions
+
+1. Get VCH VM
+2. Read back VCH configuration from VM guestinfo
+3. Delete following resources based on VCH configuration.
+ - Container VMs managed by the VCH
+   - Contianer datastore paths and resource pool path configured during installation will be used here, to detect if the VM belongs to this VCH. If yes, these VMs will be removed.
+   - Container VMs will be removed even -force is not specified
+ - volumes managed by the VCH
+   - -force option is required to delete volumes together with VCH uninstallation.
+    - Volume datastore path (sample: ds://datastore1/volume/vch1) configured during installation will be used here, to detect if the volumes are created by this VCH.
+     - If volume directory (sample: ds://datastore1/volume/vch1) is created by vic-machine create, vic-machine delete should delete this directory as well (how to know if the directory is created by vic-machine create?).
+ - images managed by the VCH
+   - Images are shared by VCHs. Currently no reference or metadata to specify which image is referenced by which VCH, so for TP3, vic-machine delete will not touch images.
```

Comment:
As our discussion today, unlikely we'll have this file directory ready before TP3, then please let me know where is current image directory while it's ready, and then will remove that directory during delete.